ludwig/data/__init__.py

```diff
@@ -0,0 +1,14 @@
+import sys
+from typing import Union
+
+try:
+    import dask.dataframe as dd
+except ImportError:
+    pass
+import pandas as pd
+
+
+if 'dask.dataframe' in sys.modules:
```

Comment:
Another way we could do this would be:

```
except ImportError:
    dd = None

...

if dd is not None:
```

But this works, too.

Reply:
Thanks for the suggestion. I prefer this as well, updated in the latest patch.